styleguide/backstop.json

```diff
@@ -111,7 +111,7 @@
     {
       "label": "topics-page",
       "url": "http://localhost:3000/?p=pages-topic",
-      "referenceUrl": "https://americanmedicalassociation.github.io/ama-style-guide-2/?p=pages-topic",
+      "referenceUrl": "https://americanmedicalassociation.github.io/ama-style-guide-2/?p=pages-topic.html",
```

Comment:
This is changing the `referenceUrl` not the normal `url` that it uses to test please update the normal `url`. This is not the right URL, the paths change a bit more significantly than just adding `.html` when opening in a new window. Please check the URL using the "Open in a new Window" link that's available when clicking the Gear at the top right hand corner of pattern lab.